packages/contracts/docs/osx/02-how-to-guides/02-plugin-development/03-non-upgradeable-plugin/01-initialization.md

```diff
@@ -1,26 +1,31 @@
 ---
-title: Initialization
+title: How to initialize Non-Upgradeable Plugins
 ---
 
-## Initializing Non-upgradeable Plugins
+## Initializing Non-Upgradeable Plugins
 
-Let's start from the beginning: Initialization. Every plugin receives and stores the address of the DAO it is associated with.
-In addition, your plugin implementation might introduce other storage variables that need to be initialized immediately after the contract was created. This is also the case for our `SimpleAdmin` plugin example the admin address.
+Every plugin should receive and store the address of the DAO it is associated with upon initialization. This is how the plugin will be able to interact with the DAO that has installed it.
+
+In addition, your plugin implementation might want to introduce other storage variables that should be initialized immediately after the contract was created. For example, in the `SimpleAdmin` plugin example, we want to store the `admin` address.
```

Comment:
I disagree. I think if we mention the PluginSetup contract now, that's when it becomes more confusing. We haven't mentioned it at all, so introducing it would require more information.

I've added additional text to explain what the `SimpleAdmin` plugin does, so it makes more sense why storing the `admin` address makes sense in this context.

Reply:
My point is about introducing this topic later, when we explain the PluginSetup. Not about introducing the PluginSetup earlier.